lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +24,14 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if type(text) is not str or len(text) == 0:
```

Comment:
use isinstance instead of `type(...) == ..`

Reply:
done